Docs.md

```diff
@@ -226,7 +226,7 @@ You should use this class for all of your Thorn tests except those which test th
 
 ### Creating Agents
 
-An Agent corresponds directly to a user that exists in the SugarCRM instance you are testing. That user *must* exist in the database before you attempt to create a user agent for it. You can use the [Thorn Fixtures API](#thorn::fixtures) to create any needed users.
+An Agent corresponds directly to a user that exists in the SugarCRM instance you are testing. All users except for the OOTB admin user *must* be created by `Fixtures.create` before you attempt to create a user agent for it. Please refer to [Thorn Fixtures API](#thorn::fixtures) to create the needed users.
```

Comment:
@rhoggSugarcrm @jcsmorais I added verbiage to this section discussing the default admin user, specifically saying that it does not need to be created.

I'm not 100% sure on the term we should use to refer to the admin user that is created by default, so I'm calling it "OOTB admin" here. If you have any better ideas, I'd like to go with that.